src/scicat_offline_ingestor.py

```diff
@@ -38,7 +38,10 @@ def build_offline_config() -> OfflineIngestorConfig:
     # with ``OnlineIngestorConfig``.
     del merged_configuration["kafka"]
 
-    return build_dataclass(OfflineIngestorConfig, merged_configuration)
+    config = build_dataclass(OfflineIngestorConfig, merged_configuration)
+    config.scicat = SciCatOptions.from_configurations(merged_configuration["scicat"])
```

Comment:
``build_dataclass`` automatically builds the sub-dataclasses as well.
So it should be not needed.
If it wasn't it's a bug so I need to fix it...